shraga_common/app/services/test_history_service.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from unittest.mock import Mock, patch
+
+from shraga_common.app.services.history_service import extract_user_org, log_interaction
+
+
+class TestExtractUserOrg:
```

Comment:
One test suite per file
And this test suite should be with the utility function anyway.